src/dataDisplay/popover/popover.jsx

```diff
@@ -63,15 +63,14 @@ const defaultProps = {
     className: undefined,
     id: undefined,
     maxWidth: undefined,
-    popperStyles: {},
     width: 250,
 };
 
 const useStyles = makeStyles((theme) => ({
     arrow: {
         color: get(theme, 'palette.common.white'),
     },
-    popper: ({ popperStyles }) => (popperStyles || {}),
+    popper: {},
```

Comment:
Now that we're not using the `popperStyles` prop anymore, can we remove its appearances inside `.\src\dataDisplay\personCoreMilestones\personCoreMilestones.jsx`

Reply:
Fixed.